share/eds/blockstore.go

```diff
@@ -0,0 +1,188 @@
+package eds
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"github.com/filecoin-project/dagstore"
+	"github.com/filecoin-project/dagstore/shard"
+	lru "github.com/hashicorp/golang-lru"
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	ipld "github.com/ipfs/go-ipld-format"
+)
+
+var _ blockstore.Blockstore = (*Blockstore)(nil)
+
+var (
+	maxCacheSize            = 100
```

Comment:
@Wondertan @renaynay what do you think I should do about this?

Reply:
Given a 30 second blocktime, this would keep almost an hour's worth of blockdata in cache. I guess that's reasonable. I wouldn't want to extend it / make it larger as I'm not sure how necessary that would be. We'd have to have good justification for it. Maybe @Wondertan has a diff idea.